airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -0,0 +1,340 @@
+version: 0.47.2
+type: DeclarativeSource
+definitions:
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_klaviyo/schemas/{{ parameters.name }}.json"
+
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: [data]
+
+  legacy_api_requester:
+    type: HttpRequester
+    url_base: https://a.klaviyo.com/api/v1/
+    path: "{{ parameters.path }}"
+    http_method: GET
+    request_headers: {}
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: '{{ config[''api_key''] }}'
+      inject_into:
+        type: RequestOption
+        inject_into: request_parameter
+        field_name: api_key
+    request_body_json: {}
+
+  revised_api_requester:
+    type: HttpRequester
+    url_base: https://a.klaviyo.com/api/
+    path: "{{ parameters.path }}"
+    http_method: GET
+    request_headers:
+      revision: "{{ parameters.revision }}"
+      Accept: application/json
+      Content-Type: application/json
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: 'Klaviyo-API-Key {{ config[''api_key''] }}'
+      inject_into:
+        type: RequestOption
+        inject_into: header
+        field_name: Authorization
+    request_body_json: {}
+
+  base_retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+
+  base_stream:
+    type: DeclarativeStream
+    primary_key:
+      - '{{ parameters.primary_key }}'
```

Comment:
As far as I can tell, this can be removed as you are already declaring primary keys in the streams below that are inheriting this.

Reply:
Once you've removed this, you should be able to run the tests and fix any other issues.